tests/kerastuner/tuners/randomsearch_test.py

```diff
@@ -11,3 +11,52 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from kerastuner.engine import multi_execution_tuner
+from kerastuner.tuners import randomsearch
+
+
+@pytest.fixture(scope='function')
+def tmp_dir(tmpdir_factory):
+    return tmpdir_factory.mktemp('randomsearch_test', numbered=True)
+
+
+def test_update_space(tmp_dir):
```

Comment:
Please add a comment describing the behavior that the test is checking.

Reply:
Done